guide/sections/part2/wis2-architecture.adoc

```diff
@@ -12,7 +12,7 @@ A WIS Centre may also operate one or more Global Services.
 
 WIS Centres shall comply with the Technical Regulations defined in the Manual on WIS (WMO No. 1060), Vol I.2.
```

Comment:
```suggestion
WIS Centres shall comply with the Technical Regulations defined in the Manual on WIS (WMO No. 1060), Vol II.
```

Reply:
WIS Centres shall comply with the Technical Regulations defined in the _Manual on WMO Information System_ (WMO-No. 1060), Volume II.